src/index.js

```diff
@@ -11,6 +10,32 @@ const pokemonImage = (pokeID, container) => {
   container.append(image);
 };
 
+const openPopup = (pokeData) => {
+  const modalBox = document.querySelector('.modal');
+  modalBox.innerHTML = `
+      <div class="popup">
+      <img src="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/${pokeData.id}.png" alt="${pokeData.name}" />
+          <h3>${pokeData.name}</h3>
+          <p>#${pokeData.id}</p>
+          <button class="closePopupBtn">Close</button>
+      </div>
+  `;
+  const popup = document.querySelector('.popup');
+  const pokeType = document.createElement('ul');
+  pokeType.innerHTML = 'Type(s)';
+  pokeType.classList.add('types');
+  popup.appendChild(pokeType);
+
+  getCardTypes(pokeData.types, pokeType);
```

Comment:
Is there a way we can modularize the popup function for best javascript practices

Reply:
I'll see. If not We'll keep it as is